posthog/temporal/data_imports/pipelines/stripe/constants.py

```diff
@@ -0,0 +1,8 @@
+ACCOUNT_RESOURCE_NAME = "Account"
+BALANCE_TRANSACTION_RESOURCE_NAME = "BalanceTransaction"
+CHARGE_RESOURCE_NAME = "Charge"
+CUSTOMER_RESOURCE_NAME = "Customer"
+INVOICE_RESOURCE_NAME = "Invoice"
+PRICE_RESOURCE_NAME = "Price"
+PRODUCT_RESOURCE_NAME = "Product"
+SUBSCRIPTION_RESOURCE_NAME = "Subscription"
```

Comment:
thanks for this change - good improvement

Reply:
BTW I tried changing this in a separate PR for easier eview, but I ended up creating conflicting changes 😞